ocamlbuild/findlib.ml

```diff
@@ -153,7 +153,7 @@ let add_atom a l = match a, l with
   | A "", _ -> l
   | _ -> a :: l
 
-let compile_flags l =
+let include_flags ?(arg="-I") l =
```

Comment:
I'd prefer the patch without the hard-to-document optional argument, but I have nothing against exposing a separate function for `-L` if that is useful.